packages/ui/src/molecules/AggregateRating/AggregateRating.test.tsx

```diff
@@ -0,0 +1,87 @@
+import { render } from '@testing-library/react'
+import { axe } from 'jest-axe'
+import React from 'react'
+
+import AggregateRating from './AggregateRating'
+
+const fillCheck = (value: number) => {
+  if (value >= 1) {
+    return 'full'
+  }
+
+  if (0 < value && value < 1) {
+    return 'half'
+  }
+
+  return 'empty'
+}
+
+describe('Breadcrumb', () => {
```

Comment:
Ooops copy & paste.
😆 

```suggestion
describe('AggregateRating', () => {
```